src/compo/mopitt_co_nc2ioda.py

```diff
@@ -0,0 +1,234 @@
+#!/usr/bin/env python3
+
+#
+# (C) Copyright 2022 UCAR
+#
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+#
+
+import sys
+import argparse
+import netCDF4 as nc
+import numpy as np
+from datetime import datetime, timedelta
+import os
+from pathlib import Path
+
+IODA_CONV_PATH = Path(__file__).parent/"@SCRIPT_LIB_PATH@"
+if not IODA_CONV_PATH.is_dir():
+    IODA_CONV_PATH = Path(__file__).parent/'..'/'lib-python'
+sys.path.append(str(IODA_CONV_PATH.resolve()))
+
+import ioda_conv_engines as iconv
+from collections import defaultdict, OrderedDict
+from orddicts import DefaultOrderedDict
+
+locationKeyList = [
+    ("latitude", "float"),
+    ("longitude", "float"),
+    ("datetime", "string"),
+]
+
+obsvars = {
+    'carbonmonoxide_total_column': 'carbon_monoxide_in_total_column',
+}
+
+AttrData = {
+    'converter': os.path.basename(__file__),
+    'nvars': np.int32(len(obsvars)),
+}
+
+DimDict = {
+}
+
+VarDims = {
+    'carbon_monoxide_in_total_column': ['nlocs'],
+}
+
+# constants
+avogadro = 6.02214076E23
```

Comment:
definitely a nitpick (so feel free to ignore) but some of these are `E` and others are `e`

Reply:
Done